_layouts/assignment.html

```diff
@@ -73,6 +73,6 @@ <h2>Submission Instructions</h2>
 	
 	{% assign temp = page.points | plus: 0 %}
 	{% if temp != total %}
-		<p>Given total number of points does not match actual total.</p>
+		<p>The given total number of points ({{ page.points }}) does not match the summed total of points from the rubric.</p>
```

Comment:
again, why is this necessary? we never show the total points, we shouldn't need this at all

Reply:
It's used on the main assignments page to avoid having to sum up rubric items there, too.